frontend/projects/upgrade/src/app/shared-standalone-component-lib/components/common-page/common-page.component.html

```diff
@@ -2,7 +2,7 @@
   <header class="header">
     <ng-content select="[header]"></ng-content>
   </header>
-  <section class="content">
-    <ng-content select="[content]"></ng-content>
+  <section class="content-container">
+    <ng-content select="[section-card]"></ng-content>
   </section>
 </div>
```

Comment:
Shouldn't the content section be like this?
```
  <section class="content">
    <ng-content select="[content]"></ng-content>
  </section>
  ```

Reply:
I think class="content" is right for styling, but we want `section-card` here because it'll get used like this:

```
<section class="content>
  <app-common-section-card section-card>Hi I'm a section card 1</app-common-section-card>
  <app-common-section-card section-card>Hi I'm a section card 2</app-common-section-card>
  <app-common-section-card section-card>Hi I'm a section card 3</app-common-section-card>
</section>
```

not like this with `content` to select multiple cards

```
<section class="content>
  <app-common-section-card content>Hi I'm a section card 1</app-common-section-card>
  <app-common-section-card content>Hi I'm a section card 2</app-common-section-card>
  <app-common-section-card content>Hi I'm a section card 3</app-common-section-card>
</section>
```